api/gin/httpServer.go

```diff
@@ -9,11 +9,11 @@ import (
 )
 
 type httpServer struct {
-	server *http.Server
```

Comment:
please change https://github.com/ElrondNetwork/elrond-eth-bridge/blob/7c86f6528dcf6362918df17f91f64cd184c7bf52/.github/workflows/golangci-lint.yml#L19
to `v1.45` in order to make the golang ci linter pass